doc/source/release_notes/release_template.rst

```diff
@@ -1,7 +1,7 @@
 :orphan:
 
-scikit-image 0.X.0 release notes
-================================
+scikit-image 0.X.0
```

Comment:
Shouldn't this get a `(date)` as well?

Reply:
Good point.

Reply:
Though, I don't think we have much use for the template now that we are using changelist. Should we just remove it (in a follow-up PR)?